website/content/docs/secrets/ad/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
-page_title: Active Directory - Secrets Engines
+page_title: Active directory secrets engine
```

Comment:
Since [Microsoft docs](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/active-directory-domain-services-overview) capitalize `Active Directory` all the time, we should respect that.